stl/inc/memory

```diff
@@ -417,7 +417,7 @@ _NoThrowFwdIt uninitialized_value_construct_n(_NoThrowFwdIt _First, const _Diff
 
 #endif // _HAS_CXX17
 
-
+#if !_HAS_CXX20

```

Comment:
`raw_storage_iterator` should probably be controlled by the usual system of coarse/fine-grained escape hatches.

Reply:
Done under `_HAS_DEPRECATED_RAW_STORAGE_ITERATOR`